tests/etc/identify_ipf_differences.py

```diff
@@ -50,26 +62,24 @@ def find_representative_bursts(important_only=False):
         'maxResults': 2000,
     }
     results = asf.search(**options)
-    bursts = sorted(get_burst_infos(results, Path.cwd()), key=lambda x: cast(datetime, x.date))
-    mid_bursts = []
-    for i in range(len(VERSIONS) - 1):
-        date1 = VERSIONS[i][1]
-        date2 = VERSIONS[i + 1][1]
-        bursts_between = [burst for burst in bursts if date1 <= cast(datetime, burst.date) < date2]
-        mid_index = int(np.floor(len(bursts_between) / 2))
-        mid_burst = bursts_between[mid_index]
-        if important_only:
-            if VERSIONS[i][2]:
-                mid_bursts.append(mid_burst)
-        else:
-            mid_bursts.append(mid_burst)
-    return mid_bursts
-
-
-def download_slcs():
-    slcs = [f'{burst.slc_granule}-SLC' for burst in find_representative_bursts()]
+    burst_infos, versions = [], []
+    for version in VERSIONS:
+        if important_only and not version.important:
+            continue
+        matching_version = [x for x in results if x.umm['PGEVersionClass']['PGEVersion'] == f'00{version.id}']
+        if len(matching_version) == 0:
+            print(f'No bursts with version {version.id} found')
+            continue
+        burst_infos.append(get_burst_infos([matching_version[len(matching_version) // 2]], Path.cwd())[0])
```

Comment:
What's the purpose of passing `matching_version[len(matching_version) // 2]` to `get_burst_infos`?

Reply:
Is that worth documenting as a comment, or do you think it's self-evident?